cabal-install/Distribution/Client/IndexUtils.hs

```diff
@@ -262,7 +262,7 @@ getSourcePackagesAtIndexState verbosity repoCtxt mb_idxState = do
             return ()
         IndexStateTime ts0 -> do
             when (isiMaxTime isi /= ts0) $
-                warn verbosity ("Requested index-state " ++ display ts0
+                info verbosity ("Requested index-state " ++ display ts0
```

Comment:
Yes, I suppose that is something we could warn about, but that's a different warning!